climada/util/api_client.py

```diff
@@ -486,3 +493,145 @@ def purge_cache(self, local_path):
+            Type of climada Hazard. If None, options of available hazard types are given
+        data_dir : str
+            directory where the files should be downoladed. Default: SYSTEM_DIR
+        """
+        if not hazard_type:
+            while True:
+                hazard_type = input("The following data types are available: "
+                                    + ", ".join(HAZ_TYPES) + ". Which one do you want to get?")
+                if hazard_type in HAZ_TYPES:
+                    break
+                else:
+                    LOGGER.error('Please give a valid value from the list provided.')
+        datasets = self._get_data(hazard_type)
+
+        hazard_list = []
+        for dataset in datasets:
+            if os.path.isfile(os.path.join(data_dir, dataset.files[0].file_name)):
+                LOGGER.info('The file already exists and it was not downloaded again.')
+            else:
+                self.download_file(data_dir, dataset.files[0])
+            hazard = Hazard()
+            hazard.read_hdf5(os.path.join(data_dir, dataset.files[0].file_name))
+            hazard_list.append(hazard)
+        hazard_concat = Hazard()
+        hazard_concat = hazard_concat.concat(hazard_list)
+        return hazard_concat
+
+    def get_exposures(self, exposures_type=None, data_dir=SYSTEM_DIR):
+        """Provides options to chose an exposures dataset, saves the file locally and open it as a climada Exposures.
+        Several countries can be given, creating an exposure combining the single countries
+        Parameters
+        ----------
+        exposures_type : str
+            Type of climada Exposures. If None, options of available exposures types are given
+        data_dir : str
+            directory where the files should be downloaded. Default: SYSTEM_DIR
+        """
+        if not exposures_type:
+            while True:
+                exposures_type = input("The following exposures types are available: "
+                                       + ", ".join(EXP_TYPES) + ". Which one would you like to get?")
+                if exposures_type in EXP_TYPES:
+                    break
+                else:
+                    LOGGER.error('Please give a valid value from the list provided.')
+        datasets = self._get_data(exposures_type)
+        exposures_list = []
+        for dataset in datasets:
+            self.download_file(data_dir, dataset.files[0])
+            if os.path.isfile(os.path.join(data_dir, dataset.files[0].file_name)):
+                LOGGER.info('The file already exists and it was not downloaded again.')
+            exposures = Exposures()
+            exposures.read_hdf5(os.path.join(data_dir, dataset.files[0].file_name))
+            exposures_list.append(exposures)
+        exposures_concat = Exposures()
+        exposures_concat = exposures_concat.concat(exposures_list)
+        return exposures_concat
+
+    def get_litpot_default(self, country=None, data_dir=SYSTEM_DIR):
```

Comment:
I think that this function is useful on top of get_exposures() as mostly people will use the default and I think it may be easier without choosing options